include/nagios.h

```diff
@@ -326,6 +326,7 @@ extern struct load_control loadctl;
 #define NOTIFICATION_DOWNTIMEEND        6
 #define NOTIFICATION_DOWNTIMECANCELLED  7
 #define NOTIFICATION_CUSTOM             8
+#define NOTIFICATION_RECOVERY           16
```

Comment:
The previous notification macros go from 0 to 8, and your new one jumps to 16. Why is that?

Reply:
mistake